src/auth/auth.service.ts

```diff
@@ -9,13 +10,18 @@ export class AuthService {
     private jwtService: JwtService,
   ) {}
 
-  async validateUser(email: string, pass: string): Promise<any> {
+  async validateUser(email: string, rawPassword: string): Promise<any> {
     const user = await this.usersService.findOneByEmail(email);
 
```

Comment:
**praise**: Using bcrypt.compare for password validation enhances security by preventing direct password comparison.